service-worker/custom-offline-page/service-worker.js

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright 2015 Google Inc. All Rights Reserved.
+ Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
If you update the copyright year, you give up the copyright before the updated year. If the code has substantially changed, the recommendation is to add a new year:

 ```suggestion
 Copyright 2015, 2019 Google Inc. All Rights Reserved.
```

Reply:
Gotcha.